### PR TITLE
Danmark.txt

### PR DESCRIPTION
**Pull request by DanniDev1**
[Denmark](https://forebears.io/denmark/forenames)

### CODE REVIEW

**Comment by Dqnns** [comment]
agreed, but you could add more and not just only your ign.

**Comment by Dqnns** [comment]
You forgot to use the script...

cleanup.py

**Review by Mytologi — COMMENTED** [review]
Added more names

**Comment by Mytologi** [comment]
> You forgot to use the script...
> 
> cleanup.py

I tried, but it didn't seem to work.

**Comment by Dqnns** [comment]
> 

Do you have python installed?

**Comment by Mytologi** [comment]
> 

Yes i do, but when i click it, it gives me a pop-up and closes fast.

**Comment by Dqnns** [comment]
image

You need to change this to a .txt file.

**Comment by Mytologi** [comment]
> image
> 
> You need to change this to a .txt file.

There we I I think

**Comment by Dqnns** [comment]
Run the file and push

**Comment by Mytologi** [comment]
> Run the file and push

Yes that's what I'm trying to do https://gyazo.com/e28830ec3a47c5c13ec4538824df8adb

**Comment by Dqnns** [comment]
You have to unzip it

**Comment by Mytologi** [comment]
> You have to unzip it

i tried that as well, didn't do the trick sadly.

**Comment by Dqnns** [comment]
Hm... Then I can't help you. Sorry. Maybe ask on the laby.net Discord.

**Review by orsou — CHANGES_REQUESTED** [review]
Why would you add names like "Asbjørn" when Minecraft doesn't even support unicode letters like "ø"? You could either exchange these letters with their latin counterparts, which in this case would be "oe" (as far as I know) or remove these names completely.

**Comment by Mytologi** [comment]
I have made an update, thanks for your suggestion.

File: src/names/Danmark.txt [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+Alfred
+Elise
+Jonathan
+Solvejg
+Kamma
+Ulrich
+Victor
+Paw
+Maj
+Kåre
```
Comment (LabyStudio) [inline src/names/Danmark.txt:459]: Please run the cleanup script to avoid impossible Minecraft names

Comment (Mytologi) [reply src/names/Danmark.txt:459]: Hi, i've tried to run the cleanup script, i'm not sure why but it's not working for me, if you're able to help me get it to work I would gladly do so, as you can see I also said this to this person here https://github.com/LabyMod/og-names/pull/9#issuecomment-996184273

**Review by Mytologi — COMMENTED** [review]
i removed a few names

**Review by Mytologi — COMMENTED** [review]
removed more names, and invalids

File: src/names/Danmark.txt [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+Freddy
+Kamilla
+Elly
+Jon
+Max
+Leo
+Helga
+Doris
+Viggo
+Maj-Britt
```
Comment (Dqnns) [inline src/names/Danmark.txt]: invalid

**Comment by Dqnns** [comment]
There are still some none "OG" names in this list...

**Comment by Mytologi** [comment]
> 

Which?

**Review by Mytologi — COMMENTED** [review]
updated

**Comment by Mytologi** [comment]
> There are still some none "OG" names in this list...

i think I removed those

**Comment by scrummer** [comment]
> > Run the file and push
> 
> Yes that's what I'm trying to do https://gyazo.com/e28830ec3a47c5c13ec4538824df8adb

@Mytologi
First unzip the repository, then start your terminal/cmd:
1. Go to your directory (e.g. `cd Downloads/og-names`)
2. Run the script from the commandline: `python ./cleanup.py`

**Review by Mytologi — COMMENTED** [review]
Hope i did it correctly this time ...

File: src/names/Danmark.txt (hunk 1 above)
Comment (Mytologi) [reply src/names/Danmark.txt]: I have done as requested of you now, i think so.

**Comment by Mytologi** [comment]
> > > Run the file and push
> > 
> > 
> > Yes that's what I'm trying to do https://gyazo.com/e28830ec3a47c5c13ec4538824df8adb
> 
> @Mytologi First unzip the repository, then start your terminal/cmd:
> 
> 1. Go to your directory (e.g. `cd Downloads/og-names`)
> 2. Run the script from the commandline: `python ./cleanup.py`

Should be done now, thanks for the tip! :D

**Comment by Dqnns** [comment]
Okay, how do I start.

The list is very inaccurate. That's because there are some names in the list that don't belong there. 

I can give many examples from this list, if you want I can edit the list for you. Just let me know.

**Comment by DanniDev1** [comment]
> 



> Okay, how do I start.
> 
> The list is very inaccurate. That's because there are some names in the list that don't belong there.
> 
> I can give many examples from this list, if you want I can edit the list for you. Just let me know.

Yeah that would be very helpful, thank you.

**Review by orsou — APPROVED** [review]
Referring to [this comment](https://github.com/LabyMod/og-names/pull/15#issuecomment-1013939277) I left on [pullrequest #15](https://github.com/LabyMod/og-names/pull/15), I would say it is up to the laby.net team to decide if all of these names are OG, the list seems clean tough and is mergeable in my opinion.

**Comment by DanniDev1** [comment]
When do they add these names to the list or approve?

**Comment by orsou** [comment]
> When do they add these names to the list or approve?

Whenever they feel like it.

**Pull request merged**